cfn.yaml

```diff
@@ -178,4 +178,22 @@ Resources:
             {
               "featureId": "epic",
               "platformId": "ios"
-            }
\ No newline at end of file
+            }
+
+  DataLakeAlertsFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      AlarmActions:
+      - !Sub arn:aws:sns:${AWS::Region}:${AWS::AccountId}:jacob-test
```

Comment:
I'm not too sure where these alerts should be sent, currently I'm the only person regularly contributing to this project...

Reply:
Now that's a good question!